content/design-systems-for-developers/react/en/document.md

```diff
@@ -156,6 +156,21 @@ First, let’s take control of the Avatar doc generation from the default. Regis
 configure(require.context('../src', true, /\.stories\.(js|mdx)$/), module);
 ```
 
+Let's add some configuration options to the addons-docs preset.
+
+```javascript
+module.exports = [
+  {
+    name: '@storybook/addon-docs/react/preset',
+    options: {
+      configureJSX: true,
+      babelOptions: {},
+      sourceLoaderOptions: null,
```

Comment:
Thanks for the PR! Couple things:

1. Just `configureJSX` is required here for MDX support, right?
2. I see another part of this file that seems to duplicate the preset. Find it here:

https://github.com/chromaui/learnstorybook.com/blob/2863cbecd87ce23c9d7c86b3b2e5750435284091/content/design-systems-for-developers/react/en/document.md#write-stories-generate-docs

Feels like the area linked above should instead be updated to include the `options` object with only the option that we need for MDX support. What do you think?

Reply:
that sounds great!
let me do the fix